src/main/windows/main.js

```diff
@@ -44,7 +43,8 @@ function init (state, options) {
     useContentSize: true, // Specify web page size without OS chrome
     width: initialBounds.width,
     webPreferences: {
-      nodeIntegration: true
+      nodeIntegration: true,
+      backgroundThrottling: false // do not throttle animations/timers when page is background
```

Comment:
Given that it was not in inside `webPreferences` I assume it was in it's default state: `true`. So I wouldn't change it here 😅